.github/workflows/ci.yml

```diff
@@ -130,7 +130,9 @@ jobs:
     name: docs-snippets
     needs: [docs-changes, ci-is-on-main-repo]
     # run on non-draft PRs with docs changes
-    if: (github.event.pull_request.draft == false && needs.docs-changes.outputs.docs == 'true')
+    if: |
+      (github.event.pull_request.draft == false && needs.docs-changes.outputs.docs == 'true') ||
+      (github.event_name == 'push' && contains(github.ref, 'refs/tags/'))
```

Comment:
This makes the conditional like `docs-checks` which did successfully run on the release tag. We want these tests to run on the release pipeline and that's what the new conditional after the `||` enables.